stable/prometheus-adapter/templates/custom-metrics-configmap.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   config.yaml: |
+    externalRules:
+{{- if .Values.externalRules.custom }}
```

Comment:
Should remove the if statement since the type of the field will change from array to string depending upon whether it's supplied or not and cause helm warnings.

Reply:
They type will not change, it's set to empty array in values.yaml for the if statement should stay same as for `rules.custom` in this file

Reply:
The `rules.custom` doesn't have a key, it was appended to the other rules if present so it was fine. Yours would've rendered as `externalRules: ""` when no rules before, but now that you changed it to put if around externalRules, it looks good. Thanks.